src/dom.js

```diff
@@ -127,6 +127,20 @@ export function removeChildren(parent) {
 
 
 /**
+ * Moves all children nodes of element "from" to element "to".
+ * @param {!Element} from
+ * @param {!Element} to
+ */
+export function moveChildren(from, to) {
+  const frag = to.ownerDocument.createDocumentFragment();
+  for (let n = from.firstChild; n; n = n.nextSibling) {
+    frag.appendChild(n);
+  }
```

Comment:
while seems to be the most appropriate loop for this (makes this bit more readable)